### DISCUSSION TITLE
Jira API Integration

### DISCUSSION THREAD

**Discussion by mdlmarkham** [FlowiseAI/Flowise#3162] [Ideas]
My team uses Jira (both Software and Service Management) to track our work - I would like to use FlowiseAI to build a chat-bot that could interface with Jira via the Atlassian API (that we are already using for the Confluence integration) and dynamically build the agenda for our morning standup meeting. This would require an analysis of the new support tickets and their severity, as well as the progress and changes on backlog items.

**Comment by abhaysahu** [comment]
Hello @mdlmarkham, I was wondering if you were able to find a solution or make any progress on this? I'd love to hear how it's going or if you faced any challenges!

**Comment by kelekci71** [comment]
Hi, would like it to hear about news as well since we are also using these technologies
